src/transformers/modeling_flax_utils.py

```diff
@@ -89,6 +89,7 @@ class FlaxPreTrainedModel(PushToHubMixin, FlaxGenerationMixin):
     base_model_prefix = ""
     main_input_name = "input_ids"
     _auto_class = None
+    _missing_keys = set()
```

Comment:
@patrickvonplaten Added `_missing_keys` attribute here to keep track if the missing keys during `from_pretrained`